### DISCUSSION TITLE
怎么监听音频流，使用客户端去处理决定回复内容

### DISCUSSION THREAD

**Discussion by GYLdaiyu** [rts-cn/rts#48] [XSwitch]
怎么监听音频流，使用客户端去处理决定回复内容

**Comment by seven1240** [comment]
我不知道你想表达什么意思  ...

**Reply by winy85691082** [reply]
应该是想实时获取通话中的音频，拿到后发给大模型接口，然后把大模型返回给的音频流直接在这个uuid里播放，用 eavesdrop 是不是可以获取到音频流 而不是保存到某个wav文件？获取到大模型接口回复的流数据后用 DTMF信令3进行三方通话可行吗？不知道freeswitch是否支持音频流的播放，以前只用过 playback播放wav文件 ，烦请老师指点

**Reply by seven1240** [reply]
我专门写过一篇收费的文章指点 :)

https://note.mowen.cn/note/detail?noteUuid=dmr4I-ZCazdSQ254LA7RU
